content.json

```diff
@@ -1742,5 +1742,52 @@
         ]
       }
     ]
-  }
+  },
+  {
+        "language": "zh",
+        "title": "Chinese Language",
+        "categories": [
+            {
+                "title": "iOS Development Blogs",
+                "slug": "development",
+                "description": "No matter if it covers Swift or Objective-C, whether it's focused on app architecture or UI coding. As long as it's related to iOS development then it belongs here.",
```

Comment:
Ideally this line would be in Chinese, could you translate it please?

Reply:
You mean the `title`, `slug` and `description ` should all be Chinese?

Reply:
I think maybe the slug should maybe stay English, although I'd be open to having that chinese as well if you think it would work well. It is also used for the filename of the OPML file though so bear that in mind.

I think the description for sure, probably the title too, and maybe the slug if you think that would be more natural for Chinese speakers.